vignettes/distributions.Rmd

```diff
@@ -26,13 +26,19 @@ The currently available parametric delay distributions are continuous probabilit
 
 ## Discretisation and adjustment for maximum delay
 
-In `epinowcast`, delays are modeled in discrete time and with an assumed maximum delay (specified via the `max_delay` argument). Therefore, the continuous delay distributions must be discretised and adjusted for the maximum delay.
+In `epinowcast`, delays are modeled in discrete time and with an assumed maximum delay (specified via the `max_delay` argument). Therefore, the continuous delay distributions must be discretised and adjusted for the maximum delay. 
 
-To discretise the continuous probability distribution into discrete probabilities $p^{\prime}_{g,t,d}$, we use its cumulative distribution function $F^{\mu_{g,t}, \upsilon_{g,t}}$ to compute
-$$p^{\prime}_{g,t,d} = \frac{F^{\mu_{g,t}, \upsilon_{g,t}}(d+1) - F^{\mu_{g,t}, \upsilon_{g,t}}(d)}{F^{\mu_{g,t}, \upsilon_{g,t}}(D)}.$$
-In words, the discrete probability mass function (pmf) is obtained from the continuous distribution by dividing it into $D$ different bins $$(0,1],\, (1,2],\, \ldots,\, (D-1,D],$$
-where $D$ is the maximum delay covered by the model. Importantly, the bins are normalized by $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$, which ensures that the $p^{\prime}_{g,t,d}$ sum to 1. Since $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$ is the probability of reporting before the maximum delay, this can also be interpreted as conditioning our distribution on the maximum delay.
+The exact form of this discretisation is complex due to the interaction between primary and secondary events. Rather than modelling this explicitly we approximate it by assuming a uniform censoring interval for each delay of 2 days. This comes from assuming daily censoring for both the primary and secondary event that make up the delay distribution and ignoring the interaction between these censoring intervals. This means that the probability of reporting a delay of $d$ days is the probability of reporting a delay of $d+1$ days or less, minus the probability of reporting a delay of $d-1$ days or less. This is then normalised by the sum of the probabilities of reporting a delay of $d$ for all delays up to some maximum delay, $D$. 
 
-Note that because of the discretisation and normalization, the discrete random variable we obtain generally does not have exactly the same moments as the original, continuous distribution.
+More rigorously, we can define this in terms of the cumulative distribution function. Let $F^{\mu_{g,t}, \upsilon_{g,t}}$ be the cumulative distribution function of the continuous probability distribution with parameters $\mu_{g,t}$ and $\upsilon_{g,t}$. Then the probability of reporting a delay of $d$ days is
+$$p_{g,t,d} = \frac{F^{\mu_{g,t}, \upsilon_{g,t}}(d+1) - F^{\mu_{g,t}, \upsilon_{g,t}}(d-1)}{F^{\mu_{g,t}, \upsilon_{g,t}}(D + 1 ) + F^{\mu_{g,t}, \upsilon_{g,t}}(D)}.$$
```

Comment:
I'm not sure I totally follow though?

Reply:
Ah no, I just mean the normalization. You wrote F^{\mu_{g,t}, \upsilon_{g,t}}(D + 1 ) + F^{\mu_{g,t}, \upsilon_{g,t}}(D) in the denominator, but I think it should be F^{\mu_{g,t}, \upsilon_{g,t}}(D) + F^{\mu_{g,t}, \upsilon_{g,t}}(D-1). This is also what you implemented, if I read your code correctly, and assuming that D means the longest delay here.

Reply:
So I have looked at the implementation again and I think the normalisation as written is correct and the issue is driven by the use of maximum delay as you note. I think this is the case because we are using $D$ here to mean the maximum observed delay in the model but due to interval censoring the actual window of possible delays we need to normalise over includes $D+1$ delays. 

In terms of phrasing... I think as written its okay but open to suggestions.

Reply:
Ah, I think now I get it. When you write

`adjusted_lcdf = lcdf - log_sum_exp(lcdf[n], lcdf[n-1]);` in your code `n=dmax=D+1`, so this is indeed what you implemented, right?

And the normalization is correct, because `sum_d=0^D (F_{d+1} - F_{d-1})` is indeed `F_{D+1} - F_{D}` (assuming F_d=0 for d<1)

Reply:
Yes exactly, I think this does highlight that some improved commenting in the code is needed to make this clear (as it took me quite a while to work out what past sam was thinking. Will make an issue.

Reply:
Made an issue (#302) to do a better job commenting on what is happening.